common/setups/rasr/gmm_system.py

```diff
@@ -245,12 +245,14 @@ def monophone_training(
             )
 
         action_sequence = meta.align_and_accumulate_sequence(
-            align_iter, 1, mark_accumulate=False, mark_align=False
+            align_iter,
+            1,
+            mark_accumulate=kwargs.pop("mark_accumulate", False),
+            mark_align=kwargs.pop("mark_align", False),
         )
         action_sequence += meta.split_and_accumulate_sequence(
             splits, accs_per_split
         ) + ["align!"]
```

Comment:
why is nothing marked here?

It seems to me that your choice of marking is somewhat specific to your setup and could profit from generalizability.

Reply:
Inside the function the accums are marked, the splits are not. This from the i6 baseline from what I know and since that was not part of what I need at the moment, I did not really touch it. But I agree at some point it would be good to also add it here.